.gitignore

```diff
@@ -1,11 +1,13 @@
 .gradle
 build/
+local-config.properties
+model/
 !gradle/wrapper/gradle-wrapper.jar
 !**/src/main/**/build/
 !**/src/test/**/build/
 
 ### IntelliJ IDEA ###
-.idea/*
```

Comment:
Maybe `.idea/` to show that it's a folder.

Reply:
I dont understand this line, i thought * would take anything in folder

Reply:
Yeah, just keep it as is. `.grade` above has the same as `.idea` now anyways.